src/main/java/seedu/address/logic/parser/AddRecipeCommandParser.java

```diff
@@ -48,10 +49,40 @@ public AddRecipeCommand parse(String args) throws ParseException {
 
         String instruction = argMultimap.getValue(PREFIX_INSTRUCTION).get();
         String recipeImage = argMultimap.getValue(PREFIX_RECIPE_IMAGE).get();
+        assert(recipeImage.length() != 0);
 
+        if (recipeImage.length() < 13) {
+            recipeImage = "images/default.jpg";
+        } else if (!recipeImage.substring(0, 6).equals("images") && !recipeImage.substring(0, 4).equals("http")) {
+            recipeImage = "images/default.jpg";
+        }
+        /*
+            String filename = "";
+            for (int i = recipeImage.length() - 1; i >= 0; i--) {
+                if (recipeImage.charAt(i) == '/') {
+                    filename = recipeImage.substring(i + 1);
+                    break;
+                }
+            }
+            URL url = new URL(recipeImage);
+            InputStream in = new BufferedInputStream(url.openStream());
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            byte[] buf = new byte[1024];
+            int n = 0;
+            while (-1 != (n = in.read(buf))) {
+                out.write(buf, 0, n);
+            }
+            out.close();
+            in.close();
+            byte[] response = out.toByteArray();
 
+            recipeImage = this.getClass().getResource("/images").getPath() + filename;
+            FileOutputStream fos = new FileOutputStream(recipeImage);
+            fos.write(response);
+            fos.close();
+        }
+         */
```

Comment:
Just curious is this the Url part?

Reply:
Yes it is haha to save into local but I realized it works without this code so I commented it out first